daffodil-lib/src/main/scala/org/apache/daffodil/lib/schema/annotation/props/ByHandMixins.scala

```diff
@@ -451,7 +451,7 @@ trait TextStandardExponentRepMixin extends PropertyMixin {
 
   lazy val textStandardExponentRep: Found = {
     val tsec = getPropertyOption("textStandardExponentCharacter")
-    val tser = getPropertyOption("textStandardExponentRep")
+    val tser = getPropertyOption("textStandardExponentRep", expressionAllowed = true)
```

Comment:
The bug mentioned several properties that needed this. Did you confirm this is the only case that's missing?